CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 ## [Unreleased][]
 
 [Unreleased]: https://github.com/chaostoolkit-incubator/chaostoolkit-google-cloud-platform/compare/0.5.0...HEAD
+### Changed - 2022-12-30
+- Renamed field max_instance_request_concurrency in create_service method. Modified init value.
+### Changed - 2022-12-19
```

Comment:
no need to set the `### Changed` header as it's already set above

Reply:
Fixed.